packages/workbox-precaching/utils/cleanupOutdatedCaches.mjs

```diff
@@ -0,0 +1,45 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+import '../_version.mjs';
+
+const SUBSTRING_TO_FIND = '-precache-';
+
+/**
+ * Can be called as part of the `activate` event to clean up incompatible
+ * precaches that were created by older versions of Workbox, by a service
+ * worker registered under the current scope.
+ *
+ * This should be safe to use as long as you don't include `substringToFind`
+ * (defaulting to `-precache-`) in your non-precache cache names.
+ *
+ * @param {string} currentPrecacheName The cache name currently in use for
+ * precaching. This cache won't be deleted.
+ * @param {string} [substringToFind='-precache-'] Cache names which include this
+ * substring will be deleted (excluding `currentPrecacheName`).
+ * @return {Array<string>} A list of all the cache names that were deleted.
+ *
+ * @private
+ * @memberof module:workbox-precaching
+ */
+export default async function(currentPrecacheName,
```

Comment:
Can we make this a named export, so it's consistent with future plans to not have default exports?

Reply:
Changed to use the `export async function cleanupOutdatedCaches` syntax. I think that's okay? Let me know if I need to switch to defining the `cleanupOutdatedCaches` function and then later doing `export {cleanupOutdatedCaches}` instead.